docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,26 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.8.11 - February 11, 2025
+
+### ​​Enhancements
+
+- Enhanced Assessment and Schema Analysis reports:
+  - Now detects the following unsupported PostgreSQL features: -
+    - Listen / Notify events
+    - Two-Phase Commit
+    - Setting compression method with COMPRESSION clause in CREATE / ALTER TABLE
+    - Create Database options for locale, collation, strategy, and OID related settings.
+- The JSON assessment report has been enhanced to include only the new assessment issue format, removing the old format that used separate fields for each issue category.
+- The import data status command now reports tables where the import has not yet started, improving visibility for bulk imports ( import data file ).
+- The assess-migration call-home payload now includes sizing and estimation information for better insights.
+- The assess-migration command now checks the source database IOPS and warns the user if it is zero.
+
+### Bug fixes
+
+- Fixed the status reported via import data status in case import is resumed for the table having CSV datafile with the header.
```

Comment:
Not sure what this means tbh, can you be clearer

Reply:
@ShivanshGahlot

Reply:
Hey @ddhodge are you talking about the guardrail checks point?
Basically earlier we used to check whether ora2pg is installed on the system even for export data with live migration. This is not required and we have removed the dependency check in this case.

Reply:
can we rewrite it better? @ShivanshGahlot

Reply:
No I'm talking about "tables that have a CSV datafile with the header" I'm not sure what that means

Reply:
So we have two options. Either a CSV datafile without a header or one with header with column names of the table etc.

Reply:
`Fixed the status reporting via import data status when resuming the import for tables with a CSV data file that includes a header row.`

Does this sounds right? @ShivanshGahlot I've tried to rephrase it

Reply:
Yeah sounds good